cmd/neutron_query_relayer/cmd/queries/utils.go

```diff
@@ -0,0 +1,41 @@
+package queries
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+)
+
+const UrlFlagName = "url"
+
+const getTimeout = time.Second * 5
+
+func get(host string, resource string) string {
+	url := host + resource
```

Comment:
Not so important (maybe it's even an overhead here, so I'm not requesting changes), but I think it's a good practice to use url builder instead of just string concatenation. It helps to avoid silly mistakes like double slash, etc.